homeassistant/helpers/entity_component.py

```diff
@@ -325,9 +325,11 @@ def async_add_entities(self, new_entities, update_before_add=False):
                    in self.platform_entities):
             return
 
-        self._async_unsub_polling = async_track_time_interval(
-            self.component.hass, self._update_entity_states, self.scan_interval
-        )
+        if self._async_unsub_polling is None:
```

Comment:
Wait, it's always none because if it's not none the previous if statement returns?